AdminWebsite/AdminWebsite/ClientApp/src/app/services/video-hearings.service.ts

```diff
@@ -451,7 +452,7 @@ export class VideoHearingsService {
     }
 
     isHearingAboutToStart(): boolean {
-        if(this.modelHearing.scheduled_date_time && this.modelHearing.status) {
+        if (this.modelHearing.scheduled_date_time && this.modelHearing.status && this.modelHearing.isConfirmed) {
```

Comment:
I think we can remove this.modelHearing.status from this now, this was just to ensure that when creating a new booking within 30 minutes you would get locked out of most pages.

I think this.modelHearing.isConfirmed does this and more ensuring it's only confirmed bookings.

Reply:
You are correct. Implemented and pushed